sql_user_check_mongodb/README.md

```diff
@@ -0,0 +1,12 @@
+# Check user permissions for MongoDB
+
+
+### Run the script
+
+Running the script will print all the databases along with total number of collections and records inside them
+Connection string is input to this script. If looks something like this 
+mongodb://username:password@hostname:port/<database_name>?authSource=admin
+
+```shell
+connection_string="mongodb://<username>:<password>@<hostname>:<port>/<database_name>?authSource=admin
+" bash run.sh```
```

Comment:
```suggestion
connection_string="mongodb://<username>:<password>@<hostname>:<port>/<database_name>?authSource=admin" bash run.sh```
```

Also this shows the connection string in plaintext. Instead of this can we ask for env variable or prompt

Reply:
done.